### ISSUE TITLE
Please update the depedencies to latest verssions and also add flutter 3.10 support

### ISSUE THREAD

**Issue by fisforfaheem** [creativecreatorormaybenot/wakelock#206]
Because no versions of wakelock match >0.6.2 <0.7.0 and every version of zego_uikit depends on wakelock ^0.6.2, every version of zego_uikit requires wakelock 0.6.2.
(1) So, because zego_uikit_prebuilt_call >=3.3.15 depends on zego_uikit ^2.5.7 and wakelock 0.6.2 depends on wakelock_windows ^0.2.0, zego_uikit_prebuilt_call >=3.3.15 requires wakelock_windows ^0.2.0.

    Because no versions of wakelock_windows match >0.2.1 <0.3.0 and wakelock_windows <0.2.1 depends on win32 ^2.0.0, wakelock_windows <0.2.1-∞ or >0.2.1 <0.3.0 requires win32 ^2.0.0.
    And because wakelock_windows 0.2.1 depends on win32 ^3.0.0, wakelock_windows <0.3.0 requires win32 ^2.0.0 or ^3.0.0.
    And because zego_uikit_prebuilt_call >=3.3.15 requires wakelock_windows ^0.2.0 (1), zego_uikit_prebuilt_call >=3.3.15 requires win32 ^2.0.0 or ^3.0.0.
    And because file_picker >=5.2.11 depends on win32 ^4.1.3, zego_uikit_prebuilt_call >=3.3.15 is incompatible with file_picker >=5.2.11.
    So, because f4zone_2022 depends on both file_picker ^5.3.0 and zego_uikit_prebuilt_call ^3.3.15, version solving failed.
exit code 1

**Comment by isaaclyman** [comment]
Duplicate of #202.

**Comment by nomoruyi** [comment]
> I recommend to override `wakelock_winodws`:
> 
> ```yaml
> dependency_overrides:
>   # Can be removed when
>   # https://github.com/creativecreatorormaybenot/wakelock/pull/203 is merged.
>   wakelock_windows:
>     git:
>       url: https://github.com/timsneath/wakelock
>       ref: win32-v5
>       path: wakelock_windows
> ```

Not working for me, I get following message:
"Because every version of wakelock_windows from git depends on wakelock_platform_interface ^0.3.0 and wakelock 0.4.0 depends on wakelock_platform_interface ^0.2.0, wakelock_windows from git is incompatible with wakelock 0.4.0.
And because no versions of wakelock match >0.4.0 <0.5.0, wakelock_windows from git is incompatible with wakelock ^0.4.0.
So, because bio_courier depends on both wakelock ^0.4.0 and wakelock_windows from git, version solving failed."

**Comment by ashemetov** [comment]
Same with win32 package - other packages require ^4.0.0.
As a workaround I made a local copy, changed win32 to ^4.0.0 and point my app onto modified package.
Hope issue will be fixed soon.

**Comment by jamesncl** [comment]
Migrate to [https://pub.dev/packages/wakelock_plus](https://pub.dev/packages/wakelock_plus)

**Comment by creativecreatorormaybenot** [comment]
I will soon discontinue this repository. Please use [wakelock_plus: ^1.1.6](https://pub.dev/packages/wakelock_plus) instead.
